src/utils/mocks.js

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { v4 } from 'uuid';
 // eslint-disable-next-line import/prefer-default-export
 export const cardMock = () => {
```

Comment:
You can significantly improve your mocks by allowing destructing `cardMock(overrides)`. That way you don't have to assign values to individual pieces in your test (creating a mess in your test files).

```
return {
   ...mockDeck,
   ...overrides,
};
```